src/core/pdf/popplerapi/PopplerGlibPage.h

```diff
@@ -27,7 +27,8 @@ class PopplerGlibPage: public XojPdfPage {
     double getWidth() const override;
     double getHeight() const override;
 
-    void render(cairo_t* cr, bool forPrinting = false) override;  // NOLINT(google-default-arguments)
+    void render(cairo_t* cr) const override;
+    void renderForPrinting(cairo_t* cr) const override;
```

Comment:
Could you add a doc comment explaining the difference between these two?

Reply:
Will do, although I am no sure myself of exactly what the difference is. See poppler [doc](https://poppler.freedesktop.org/api/glib/poppler-Poppler-Page.html#poppler-page-render-for-printing).
I possibly relates to issues like #3692, #3177 or #1150